SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -119,6 +127,15 @@ public let DEFAULT_DISK_CACHE_MAX_SIZE_BYTES: Int64 = 25 * 1024 * 1024
      Enable NetworkInstrumentation span creation for https calls.
      */
     @objc public var networkInstrumentation: Bool = true
+     /**
+     Threshold, in milliseconds, from which to count a rendered frame as slow.
+     */
+    @objc public var slowFrameDetectionThresholdMs: Double = 16.7
```

Comment:
Would it make sense to make this dynamic based on the [max frames per second](https://developer.apple.com/documentation/uikit/uiscreen/2806814-maximumframespersecond) that the screen can render?

Reply:
Hi Pablo - it's an interesting idea.  Let's get this PR back into shape, merge it, and then we can look at pursuing this as an additional tweak after.